src/test/java/com/automationPractice/pages/MainPage.java

```diff
@@ -13,6 +13,10 @@ public MainPage(WebDriver driver){
     By Module_Editorial = By.id("editorial_block_center");
     By Tittle_MainPage = By.xpath("//div[@id='editorial_block_center']//h1");
     By BTN_ContactUs = By.id("contact-link");
+    By txt_search = By.id("search_query_top");
+    By btn_search=By.xpath("//button[@name='submit_search']");
+    By Form_principal = By.id("center_column");
+    By txt_result= By.xpath("//span[@class='lighter']");
```

Comment:
do you try used By.classname instead of By.xpath?

Reply:
I replaced the locator to use by.className